### ISSUE TITLE
Component hydration fails

### ISSUE THREAD

**Issue by meigo** [Elderjs/template#14]
With latest template using elderjs v1 loading of hydrated component fails
```console
s.min.js:1 Uncaught (in promise) Error: svelte/entryClock-2e8914b9.js, http://localhost:5000/ (SystemJS https://git.io/JvFET#8) at s.min.js:1
```

Reason is backslash in beginning of import path in index.html
```js
...
System.import('\svelte/entryClock-2e8914b9.js').then(({ default: App }) => {
...
```

Probably a Windows specific quirck.

**Comment by nickreese** [comment]
@meigo pushed a fix in v1.0.4.

Thanks.

**Comment by meigo** [comment]
Beautiful, thank you.
